pkg/controller/controller.go

```diff
@@ -344,11 +344,11 @@ func getEmail(r *http.Request) (string, error) {
 }
 
 // GetCatagoriesDisplayData takes validators as input and returns UIData for display. If there's an error during the GetUIData RPC call, it will be thrown.
-func getCatagoriesDisplayData(validators map[string]jvspb.Validator) (map[string]*jvspb.UIData, error) {
-	displayData := make(map[string]*jvspb.UIData)
+func getCatagoriesDisplayData(validators map[string]jvspb.Validator, ctx context.Context) (map[string]*jvspb.UIData, error) {
```

Comment:
Context should always be the _first_ parameter to a function.


```suggestion
func getCatagoriesDisplayData(ctx context.Context, validators map[string]jvspb.Validator) (map[string]*jvspb.UIData, error) {
```

Reply:
Updated in the latest version. Thank you.